spec/util.spec.js

```diff
@@ -107,7 +107,7 @@ describe('util.fetchFile:', function() {
             // options
             expect(instance.fetchRequest.url).toEndWith(options.url);
             expect(instance.fetchRequest.cache).toEqual('default');
-            expect(instance.fetchRequest.credentials).toEqual('same-origin');
+            expect(instance.fetchRequest.credentials).toEqual('omit');
```

Comment:
i don't remember this test failing before..

Reply:
It is failing on my machine when the expectation is that `instance.fetchRequest.credentials` is `same-origin`, but succeeds on both of your CI pipelines when it is `same-origin`. So I've undone this change. I'm unsure what the specific intent behind this test is; it may be desirable to remove it entirely if it is unreliable across platforms.

Reply:
> it may be desirable to remove it entirely if it is unreliable across platforms.

sounds good.